cypress/integration/adminFlows/config/authenticationSection.spec.js

```diff
@@ -43,15 +47,16 @@ describe('Authentication Section', () => {
 
           cy.findByText('Successfully updated settings.').should('be.visible');
 
-          cy.get('@authSectionForm').findByText('Authentication').click();
-
           cy.get('@authSectionForm')
-            .findByLabelText('Invite-only mode')
+            .findByRole('heading', { name: 'Authentication' })
+            .click();
+          cy.get('@authSectionForm')
+            .findByRole('checkbox', { name: 'Invite-only mode' })
             .should('be.checked');
 
           // Ensure that none of the authentication providers are enabled.
           cy.findByLabelText('Email enabled').should('not.be.visible');
-          cy.findByLabelText('Facebook enabled').should('not.be.visible');
+          cy.findByLabelText('Facebook enabled').should('to.be.visible');
```

Comment:
We enable Facebook [here](https://github.com/forem/forem/blob/4683db1b8ac188c7c6cb362e0795153b419aff27/cypress/integration/adminFlows/config/authenticationSection.spec.js#L19), but were expecting "Facebook enabled" to not be visible in the spec, causing a failure. That said, this change reflects the fact that we _do_ enable Facebook and because of that, this _should_ be visible. Someone please jump in here if I'm wrong and I'll go back to debugging. 😅 

cc @citizen428

Reply:
@juliannatetreault 

1. The line you highlighted enables Facebook so we can later meaningfully assert that it is no longer enabled.
2. The function inside `then` activates invitation-only mode, which should disable all auth providers. 
3. The `not.be.visible` assertion is correct we should not change the spec.

Reply:
You're spot on with this, @citizen428! I've figured out the failure and have the e2e test passing locally now.